test/framework/src/main/java/org/opensearch/test/OpenSearchIntegTestCase.java

```diff
@@ -2070,7 +2070,8 @@ protected boolean addMockGeoShapeFieldMapper() {
      * @return boolean.
      */
     protected boolean addMockTelemetryPlugin() {
-        return true;
+        // setting to false until https://github.com/opensearch-project/OpenSearch/issues/12615 is resolved
+        return false;
```

Comment:
How does nothing fail with this set to false? I would have expected some of the telemetry-focused tests to require this?

Reply:
Going to move this conversation to https://github.com/opensearch-project/OpenSearch/issues/12615 so we can move with this.